applications/GeoMechanicsApplication/tests/submerged_construction_of_excavation/common/1_Initial_stage.json

```
@@ -0,0 +1,351 @@
+                    "is_fixed": [
+                        true,
+                        false,
+                        false
+                    ],
+                    "value": [
+                        0.0,
+                        0.0,
+                        0.0
+                    ],
+                    "table": [
+                        0,
+                        0,
+                        0
+                    ]
+                }
+            },
+            {
+                "python_module": "apply_vector_constraint_table_process",
+                "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+                "process_name": "ApplyVectorConstraintTableProcess",
+                "Parameters": {
+                    "model_part_name": "PorousDomain.Bottom_Edge",
+                    "variable_name": "DISPLACEMENT",
+                    "active": [
+                        true,
+                        true,
+                        false
+                    ],
+                    "is_fixed": [
+                        true,
+                        true,
+                        false
+                    ],
+                    "value": [
+                        0.0,
+                        0.0,
+                        0.0
+                    ],
+                    "table": [
+                        0,
+                        0,
+                        0
+                    ]
+                }
+            },
+            {
+                "python_module": "apply_scalar_constraint_table_process",
+                "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+                "process_name": "ApplyScalarConstraintTableProcess",
+                "Parameters": {
+                    "model_part_name": "PorousDomain",
+                    "variable_name": "WATER_PRESSURE",
+                    "is_fixed": true,
+                    "table": 0,
+                    "fluid_pressure_type"  : "Hydrostatic",
+                    "gravity_direction"    : 1,
+                    "reference_coordinate" : 18.0,
+                    "specific_weight": 10000.0
+                }
+            },
+            {
+                "python_module": "assign_master_slave_constraints_to_neighbours_process",
+                "kratos_module": "KratosMultiphysics",
+                "process_name": "AssignMasterSlaveConstraintsToNeighboursUtility",
+                "Parameters": {
+                    "model_part_name": "PorousDomain.porous_computational_model_part",
+                    "variable_names": [
+                        "DISPLACEMENT",
+                        "WATER_PRESSURE"
+                    ],
+                    "minimum_number_of_neighbouring_nodes": 2,
+                    "reform_constraints_at_each_step": true,
+                    "master_model_part_name": "PorousDomain.Master_side_1",
+                    "slave_model_part_name": "PorousDomain.Slave_side_1",
```

Since we have only one master side and only one slave side, I feel we can leave out the numbers here:
```suggestion
                    "master_model_part_name": "PorousDomain.Master_side",
                    "slave_model_part_name": "PorousDomain.Slave_side",
```

---

Done